src/lib/undux/GDStore.js

```diff
@@ -67,7 +67,8 @@ const initialState: State = {
   feeds: [],
   currentFeed: undefined,
   feedLoading: false,
-  inviteCode: undefined,
+  inviteLevel: undefined,
+  invitePointsAmount: undefined,
```

Comment:
```suggestion
  inviteCode: undefined,
  inviteLevel: undefined,
```

Reply:
1. why inviteCode was removed ? isn't it still used somewhere ?
2. what us invitePointsAmount ? i see no other reference to this GD store value in your PR

Reply:
my bad, i overwrited invite Code instead of invite Points Amount